tianshou/env/pettingzoo_env.py

```diff
@@ -55,7 +55,9 @@ def __init__(self, env: BaseWrapper):
 
         self.reset()
 
-    def reset(self) -> dict:
+    def reset(self, seed: Optional[int] = None) -> dict:
+        if seed is not None:
+            self.env.seed(seed)
```

Comment:
Has PettingZoo upgraded their API in the newest release?

Reply:
Also, you didn't change the reset's return value to `Union[obs, Tuple[obs, info]]`. That's a big issue.

Reply:
> Changing the return value to Union[obs, Tuple[obs, info]] everywhere in this repo is part of the additional work that needs to be done at some point. I can include it in this PR if desired.

I fix this in the last PR (#623) since PettingZoo==0.18.1 introduces this API.

Reply:
updated return value to `Union[obs, Tuple[obs, info]]`